lexicon/tests/providers/test_misaka.py

```diff
@@ -0,0 +1,20 @@
+"""Integration tests for Misaka.IO"""
+from unittest import TestCase
+
+from lexicon.tests.providers.integration_tests import IntegrationTestsV2
+
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from define_tests.TheTests
+class MisakaProviderTests(TestCase, IntegrationTestsV2):
+    """TestCase for Misaka.IO"""
+
+    provider_name = "misaka"
+    domain = "misaka-dns-test.stream"
+
+    def _filter_headers(self):
+        return [
+            "Authorization", "Set-Cookie",
+            "Date", "X-Misaka-Debug", "X-Request-Id", "X-Served-By", "CF-RAY", "cf-request-id",
```

Comment:
I think `Date` can be kept, right?

Reply:
Yep, let me remove it and rebuild the fixtures.